src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -101,14 +104,47 @@ public Pose2d arrayToPose(double[] array){
         return new Pose2d(array[2], -array[0], new Rotation2d(Math.toRadians(-array[4])));
     }
 
-    public Pose2d convertCameraSpaceToRobotSpace(Pose2d poseInCameraSpace){ 
-        //TODO: explain the order of this and why the camera pose from robot center is being transformed by the pose in camera space
-        Transform2d transform = new Transform2d(poseInCameraSpace.getTranslation(), poseInCameraSpace.getRotation());
-        Pose2d cameraPoseFromRobotCenter = new Pose2d(Constants.LIMELIGHT_FORWARD_OFFSET, -Constants.LIMELIGHT_SIDE_OFFSET, new Rotation2d(Math.toRadians(Constants.LIMELIGHT_YAW_OFFSET)));
-        Pose2d robotSpacePose = cameraPoseFromRobotCenter.transformBy(transform);
-        System.out.println(robotSpacePose);
-        return robotSpacePose;
-    }
+    public Pose3d arrayToPose3d(double[] array){
```

Comment:
Some comment about how this converts from limelight's wacky coordinate system to FRC's